tests/python/rest_api/test_tasks.py

```diff
@@ -1012,6 +1012,71 @@ def test_datumaro_export_without_annotations_includes_image_info(
                     if "size" in related_image:
                         assert tuple(related_image["size"]) > (0, 0)
 
+    @pytest.mark.usefixtures("restore_db_per_function")
+    @pytest.mark.usefixtures("restore_redis_ondisk_per_function")
+    @pytest.mark.parametrize(
+        "format_name, num_frames, frame_step, expected_frames",
+        [
+            ("Datumaro 1.0", 100, 5, 20),
+            ("COCO 1.0", 100, 5, 20),
+            ("CVAT for video 1.1", 100, 5, 20),  # no remainder
+            ("CVAT for video 1.1", 97, 5, 20),  # prime
+            ("CVAT for video 1.1", 97, 2, 49),
+            ("CVAT for video 1.1", 100, 3, 34),  # three
+            # we assert that expected frames are ceil(frames / step)
+        ],
+    )
+    def test_export_with_non_default_frame_step(
+        self,
+        # fixtures
+        tmp_path: Path,
+        admin_user: str,
+        # params
+        format_name: str,
+        num_frames: int,
+        frame_step: int,
+        expected_frames: int,
+    ):
+        # parameter validation
+        assert expected_frames == math.ceil(num_frames / frame_step), "Test params are wrong"
+
+        spec = {
+            "name": f"test_video_frames_in_{format_name}_after_export",
+            "labels": [{"name": "goofy ahh car"}],
+        }
+
+        data = {
+            "image_quality": 70,
+            "client_files": [generate_video_file(num_frames)],
```

Comment:
Shouldn't image tasks be checked as well?

Reply:
From what I understand, images and videos are processed with the same code, the issue was with the format only
I can add a parametrized test for images, too, if it's necessary